packages/cli/src/databases/migrations/mysqldb/1695128658538-AddWorkflowMetadata.ts

```diff
@@ -0,0 +1,13 @@
+import type { MigrationContext, ReversibleMigration } from '@db/types';
+
+export class AddWorkflowMetadata1695128658538 implements ReversibleMigration {
```

Comment:
We can use queryRunner.addColumn so we do not need to replicate the same migration 3 times. See the example in the MFA migration in the common folders

Reply:
ah nice thanks